bower.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "transac",
+  "version": "0.1.0",
+  "authors": [
+    {
+      "name": "Xaun Lopez",
+      "email": "xaunlopez@gmail.com"
+    }
+  ],
+  "main": [
+    "dist/transac.js",
+    "dist/transac.less"
+  ],
+  "ignore": [
+    "src",
+    "test",
+    "gulpfile.js",
+    "karma-*.conf.js",
+    "**/.*"
+  ],
+  "dependencies": {
+    "angular": "~1.6.0",
+    "angular-animate": "~1.6.0",
+    "angular-bootstrap": "2.3.2",
+    "angular-toastr": "1.7.0",
+    "bootstrap": "^3.3.6",
+    "lodash": "~3.10.1",
+    "font-awesome": "~4.7.0",
+    "moment": "~2.11.2",
```

Comment:
You should use https://github.com/urish/angular-moment, it is an angular wrapper.